sources/packages/api/src/database/entities/student-user.model.ts

```diff
@@ -0,0 +1,34 @@
+import { Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from "typeorm";
+import { RecordDataModel } from ".";
+import { ColumnNames, TableNames } from "../constant";
+import { Student } from "./student.model";
+import { User } from "./user.model";
+
+/**
+ * Students and users relationships, current and past ones.
+ * Every time that a student/user association changes this
+ * table will receive a new record to keep the audit.
+ */
```

Comment:
Student account application model is not created and referenced here. I guess that must be on upcoming PR.

Reply:
Yes, upcoming one 👍